File: chocolate/choco.py
# Starter Code
class Solution:
    def find_min_diff(self,arr,n,k):
        chocolate_distribution=0
        # Write your code here without removing the existing code
        # the variable 'arr' contains array of integers of size n and k is an integer.
        # modified the chocolate_distribution 'counter' and return it.
        return chocolate_distribution


if __name__ == '__main__':
    n = int(input())
    str = input()
    arr = [int(int(i)) for i in str.strip('[]').split(',')]
    x = Solution()
    k= int(input())
    ans=x.find_min_diff(arr,n,k)
    print(ans)

File: chocolate/choco.java

	
	public static void main(String[] args)
    {
			Scanner sc=new Scanner(System.in);
            int n = sc.nextInt(); 
			int []arr= Arrays.stream(sc.next().split("[,\\[\\]]")).filter(w-> !w.equals("")).mapToInt(Integer::parseInt).toArray();
		    Solution g=new Solution();
            int k = sc.nextInt(); 
            int ans=g.find_min_diff(arr,n,k);  
            System.out.println(ans);
             
          
	}
}
class Solution
{
	public static int find_min_diff(int arr[],int n,int k)
        {
            int chocolate_distribution = -1;
			//Write your code here without removing the existing code
	        //the variable 'arr' contains array of integers of size n and an integer k.
	        //modified the integer variable 'chocolate_distribution' and return it.
			return chocolate_distribution;
        }
}